src/main/java/com/google/sps/utility/ServletUtility.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utility;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+
+/** Contains methods for common servlet operations */
+public final class ServletUtility {
+  public static final String APPLICATION_NAME = "PUT NAME HERE";
+
+  private ServletUtility() {}
+
+  /**
+   * Helper method to get a cookie from an HttpServletRequest
+   *
+   * @param request HttpServletRequest that contains desired cookie
+   * @param cookieName name of desired cookie. Case sensitive
+   * @return Cookie if found, null if not found or if duplicates present
+   */
+  public static Cookie getCookie(HttpServletRequest request, String cookieName) {
+    List<Cookie> cookies =
+        Arrays.stream(request.getCookies())
+            .filter((Cookie c) -> c.getName().equals(cookieName))
+            .collect(Collectors.toList());
+
+    if (cookies.isEmpty()) {
+      System.out.println("Cookie not found");
+      return null;
+    }
+
+    // If more than one cookies are found, it is ambiguous as to which one to return.
+    // This is unexpected - duplicate cookies are usually blocked by the browser (especially
+    // when they are user set).
+    if (cookies.size() > 1) {
+      System.out.println("Duplicate cookie");
+      for (int i = 0; i < cookies.size(); i++) {
+        System.out.println(String.format("Cookie %d: %s", i + 1, cookies.get(i).getValue()));
```

Comment:
Nit 1: `System.out.printf` lets you format the string
Nit 2: `list.forEach` to avoid the explicit loop

Reply:
Nit 1: Cool! Resolved. 
Nit 2: I did initially want the index, but I think the new log is better and cleaner. Thanks for the suggestion!